Tutorials/Starter-kit/Lessons/3-Add-Open-Graph/summary.md

```diff
@@ -1,14 +1,13 @@
 # Add Open Graph - Step 4
 
-All done, great job! Now test out if it works. Try adding it to more document types. Remember this is just one way of adding this functionality. You might want additional Open Graph tags or the properties to be on a different tab (maybe Navigation & SEO).
+All done, great job! Now test out if it works. Try adding it to more document types. Remember this is just one way of adding this functionality. You might want additional Open Graph tags or the properties to be on a different tab (e.g. *Navigation* or *SEO*).
 
 ## Summary
 In this lesson you you've learned:
```

Comment:
I'm thinking if we're tidying up the grammars here, could the summary text become:

In this lesson you have learned:

?

Reply:
Ha - that's ridiculous! That's the exact line that prmpted me to click the *Edit* link in the first place 😆 

I'll get right on it...